src/mlmc/sampling_pool_pbs.py

```diff
@@ -16,21 +16,20 @@ class SamplingPoolPBS(SamplingPool):
     JOBS_DIR = "jobs"
     LEVEL_SIM_CONFIG = "level_{}_simulation_config"  # Serialized level simulation
     JOB = "{}_job.sh"  # Pbs process file
+    JOBS_COUNT = "jobs_count.txt" # Contains current number of jobs which is also job unique identifier
 
-    def __init__(self, work_dir, job_weight=200000, job_count=0):
+    def __init__(self, work_dir, job_weight=200000, force=False):
```

Comment:
I don't think we need a clean flag. Because in case of multiprocess pool there are no auxiliary files which are process dependent, all is mainly related to simulation.